pkg/pow/miningTasker.go

```diff
@@ -184,7 +326,7 @@ func (mt *MiningTasker) GetWork() (*Work, bool) {
 		return nil, false
 	}
 	mt.currChallenge = newChallenge
-	return &Work{Challenge: newChallenge, PublicAddr: mt.pubKey[2:], Start: uint64(rand.Int63()), N: math.MaxInt64}, instantSubmit
+	return &Work{Challenge: newChallenge, PublicAddr: mt.accounts[0].Address.String(), Start: uint64(rand.Int63()), N: math.MaxInt64}, instantSubmit
```

Comment:
again based on my previous comments and the problem we are trying to solve, there should be no notion of `instantSubmit` in the tasker. Its only responsibility is to receive and send tasks.